client/components/AllProducts.js

```diff
@@ -0,0 +1,33 @@
+import React, {Component} from 'react'
+import {connect} from 'react-redux'
+
+class AllProducts extends Component {
+  componentDidMount() {
+    // Uncomment line below when we have a thunk imported from the store named fetchProducts.
+    // this.props.fetchProducts();
+  }
+
+  render() {
+    const {products} = this.props
+
+    return (
+      <div>
+        {products ? (
+          <div>Will map over all the products here.</div>
```

Comment:
Placeholder code is ok especially in the early prototyping stages, but my personal bias would be to at least make this dynamic since you are pulling from `props` – e.g. `There are { products.length } products` or similar. Also, `div` is not semantic, I'd prefer `p`. I know you're going to replace this down the line, but it's good to be in the habit of reaching for the most semantically appropriate HTML tag available.